### DISCUSSION TITLE
[question] Can I help improve the documentation and storybook?

### DISCUSSION THREAD

**Discussion by NtsCiccio** [couds/react-bulma-components#336] [Q&A]
Hi, I was wondering how can I help improve the documentation and storybook?

**Comment by kennethnym** [comment]
Hi @NtsCiccio we have created a brand new documentation site for v4 on [https://react-bulma.dev/en/storybook](https://react-bulma.dev/en/storybook). Is this what you're looking for?

**Comment by kjschu** [comment]
Are we able to access the code to Storybook site? Then we can help fix typos, add/update example code, etc.

**Comment by kennethnym** [comment]
It's in the same repo :)

**Comment by NtsCiccio** [comment]
> Are we able to access the code to Storybook site? Then we can help fix typos, add/update example code, etc.

that's what i mean.

> It's in the same repo :)

cool so i can make PR if want contribute to the documentation

**Comment by kennethnym** [comment]
Feel free :) really appreciate it
